_partials/_livesync-limitations.md

```diff
@@ -0,0 +1,5 @@
+* The Schema is not migrated by Livesync, you use pg_dump/restore to migrate schema
```

Comment:
> * The Schema is not migrated by Livesync, you use pg_dump/restore to migrate schema

This is not applicable on the web console, the hosted one migrates the initial schema.

Reply:
Now in terminal section only.